src/main/java/Oak/controller/TerminalController.java

```diff
@@ -0,0 +1,54 @@
+package Oak.controller;
+
+import Oak.exceptions.InvalidInputException;
+import Oak.feedback.FeedbackService;
+import Oak.feedback.enums.CommandEnum;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * The type Terminal controller.
```

Comment:
Could the class description be more descriptive, maybe mention what the terminal controller does?

Reply:
fixed!